src/server/saved_objects/client/lib/parse_es_doc.js

```diff
@@ -0,0 +1,16 @@
+import { get } from 'lodash';
+
+export function parseEsDoc(doc, overrides = {}) {
```

Comment:
Should we add a jsdoc on this? Fairly general name, so just to make the intention clear